std/testing/asserts.ts

```diff
@@ -243,6 +243,64 @@ export function assertStrictEq(
   throw new AssertionError(message);
 }
 
+/**
+ * Make an assertion that `actual` greater than `expected` value. If not
+ * then thrown.
+ */
+export function assertToBeGreaterThan(
```

Comment:
How about `assertGreater()` and `assertLess()`?

Reply:
hi @nayeemrmn 
I will implement `assertGreater` and `assertLess` in the future

Reply:
I think assertGreater and assertLess is more meaningful .